src/js/popup.js

```diff
@@ -58,6 +61,29 @@
       alertListContainer.addClass('is-hidden');
   }
 
+  function getNowValue() {
+    nowValueService.currentValue()
+      .then(updateNowAndVariation);
+  }
+
+  function updateNowAndVariation(response){
```

Comment:
Name this function as `onGetUSDCurrentValueSuccess` and let this function just calling other two functions:
``` javascript
function onGetUSDCurrentValueSuccess(response){
  setupUSDCurrentValue(response.dolar.cotacao);
  setupUSDCurrentValueVariation(response.dolar.variacao);
}
```